### PR TITLE
Add query for drug -> drug targets

### PR DESCRIPTION
**Pull request by justaddcoffee**
Adding sparql query for drug -> drug targets per convo with @callahantiff 

@matentzn could you have a look at this sparql query and 1) confirm it's okay and 2) maybe give some advice about why it's timing out? It should return at most 30k things, so it doesn't seem too demanding

### CODE REVIEW

File: queries/sparql/query-07-drug-to-drug-target.rq [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+=+ endpoint http://kg-hub-rdf.berkeleybop.io/blazegraph/sparql
+
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix bl: <https://w3id.org/biolink/vocab/>
+SELECT ?drug ?druglab ?target ?targetlab WHERE
+{
+  ?drug ?pred ?target .
+  ?drug bl:category bl:Drug .
+  ?target bl:category bl:Protein; <https://www.example.org/UNKNOWN/ncbi_taxid> "9606"^^xsd:string .
```
Comment (matentzn) [inline queries/sparql/query-07-drug-to-drug-target.rq:12]: We really should get rid of taxon ids referenced as strings.. Very bad modelling. Should be:

```
?target bl:category bl:Protein; <https://www.example.org/UNKNOWN/ncbi_taxid> <http://purl.obolibrary.org/obo/NCBITaxon_ 9606> .
```

Comment (justaddcoffee) [reply queries/sparql/query-07-drug-to-drug-target.rq:13]: Agreed, but I think this will require a change to KGX

**Comment by justaddcoffee** [comment]
This query now works, but only yields 2,082 rows. Some proteins might not be classified as 9606/human, and/or some drug or proteins might have >1 category

**Comment by justaddcoffee** [comment]
Now at 3,474 drug -> drug target edges, by casting a wider net for drug and target bl categories (bl:Drug | bl:ChemicalSubstance for drug categories, and bl:Protein | bl:Gene | bl:NamedThing for target categories)

**Comment by justaddcoffee** [comment]
3,372 drug -> target edges when I remove targets with catefory of bl:NamedThing, which seems to broad. 

@callahantiff could we chat briefly at some point before Thursday? I think that this query will work for your purposes, but want to confirm

**Comment by callahantiff** [comment]
> 3,372 drug -> target edges when I remove targets with catefory of bl:NamedThing, which seems to broad.
> 
> @callahantiff could we chat briefly at some point before Thursday? I think that this query will work for your purposes, but want to confirm

Absolutely! Will slack you about a time.

**Comment by justaddcoffee** [comment]
@callahantiff - just a quick note that this SPARQL query for drug -> drug target has moved here:
https://github.com/Knowledge-Graph-Hub/kg-covid-19/blob/master/queries/query-07-drug-to-drug-target.rq
(b/c of [this PR](#396))

I don't think you were actually using this rq file - just making sure things don't break on your end

**Comment by callahantiff** [comment]
> @callahantiff - just a quick note that this SPARQL query for drug -> drug target has moved here:
> https://github.com/Knowledge-Graph-Hub/kg-covid-19/blob/master/queries/query-07-drug-to-drug-target.rq
> (b/c of [this PR](#396))
> 
> I don't think you were actually using this rq file - just making sure things don't break on your end

Thanks @justaddcoffee - I made sure to update things on my end!

**Pull request merged**